starfish/codebook/codebook.py

```diff
@@ -514,9 +510,11 @@ def decode_metric(
             norm_order: int, metric: str='euclidean'
     ) -> IntensityTable:
         """
-        Assigns intensity patterns that have been extracted from an :py:class:`ImageStack` and
-        stored in an :py:class:`IntensityTable` by a :py:class:`SpotFinder` to the gene targets that
-        they encode.
+        Assigns intensity patterns that have been extracted from an
+        :py:class:`~starfish.imagestack.imagestack.ImageStack` and
+        stored in an :py:class:`~starfish.intensity_table.intensity_table.IntensityTable` by a `SpotFinder
+        <https://spacetx-starfish.readthedocs.io/en/latest/api/spots/detection/detection.html>`_
```

Comment:
a sphinx cross-reference here will be more robust in the event of changing urls, rendering in PDF, or a re-org of the docs https://www.sphinx-doc.org/en/1.7/markup/inline.html#cross-referencing-arbitrary-locations